src/components/crop-content/CropContent.tsx

```diff
@@ -49,46 +54,54 @@ export default class CropContent extends React.Component<CropContentProps> {
 
   componentDidMount() {
     // after first render check if component needs toggle
-    this.handleOverflow();
+    this.computeState();
   }
 
   setRef = (e: HTMLDivElement) => {
     if(e){
       this.containerElRef = e;
       this.mutationObserver.observe(e, {childList:true, subtree:true , characterData:true});
-      this.handleOverflow();
+      this.computeState();
     }
   }
 
   componentDidUpdate(prevProps) {
     if (prevProps.cropHeight !== this.props.cropHeight) {
-      this.handleOverflow();
+      this.computeState();
     }
   }
   componentWillUnmount(){
     this.mutationObserver.disconnect();
   }
 
-  handleOverflow() {
+  computeState() {
+    const nextState: CropContentState = {...this.state};
     if (this.state.collapsed) {
       const hasOverflow =
         this.containerElRef &&
         this.containerElRef.scrollHeight - this.containerElRef.offsetHeight > 1;
       this.setState({ hasOverflow });
```

Comment:
It seems you can remove this line : `this.setState({ hasOverflow });`

Reply:
yup forgot it :D